bogenliga/bogenliga-application/src/main/resources/db/migration/LOCAL/V59__alter__table__spotter__implementierung.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE tablet_session ADD COLUMN access_token DECIMAL(19,0) DEFAULT 0;
```

Comment:
Damit macht ihr die Änderung/Anpassung an der Tabelle in der Datenbank nur lokal! Die Produktive DB wird davon nicht beeinflusst.
Legt bitte ein Skript unter /all an, wenn ihr wollt, dass die Änderungen auch auf der produktiven DB ziehen. 

evtl. braucht ihr dann zwei Skripte (eins für lokal für den Insert rolle_recht und eins für die alter table unter all).